install_files/ansible-base/roles/tor-hidden-services/tasks/install_tor.yml

```diff
@@ -1,35 +1,35 @@
 ---
-- name: Install Tor project GPG signing key.
+- name: Remove Tor project GPG signing key.
   apt_key:
-    state: present
-    data: "{{ lookup('file', 'tor-signing-key.pub') }}"
+    state: absent
+    id: A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89
```

Comment:
I think this removal logic should also be part of the effort of #2764 but I'm not blocking this marge on that . Good due diligence on removing it here thru ansible for now 👍